qml/MainMenu.qml

```diff
@@ -32,7 +32,7 @@ Item {
 
         // Transform arguments to Array and remove connectionTypeEnum from it
         var nextArgs = Array.prototype.slice.call(arguments).slice(1)
-        ping.connectLink(connectionTypeEnum, nextArgs)
+        ping.tryConnectLink(connectionTypeEnum, nextArgs)
```

Comment:
I'm not sure if this patch is actually appropriate. I think if the user is entering an explicit connection, autodetect should be disabled.

Reply:
In noise links, the communication may fail and will not try to detect again, this will make the user experience not clear when adding a URL link that can fail n times.

Reply:
The noise does not matter, that is outside the scope of this application.

If I want to connect to udp ip:port specifically, and I explicitly specify it, that is what we should do!

If the user enters a udp or serial configuration here, we should not _detect_ on that configuration, we should skip detect, and just _connect_ on that configuration, whether there is noise, garbage, or nothing. The user said to do it, so do it. This makes the user experience clear.

Reply:
The implication if the user interacts here is that the user knows where/how the sensor is connected. Therefore we do not need to detect.

Reply:
I agree with you, I will do further tests with UDP to take a closer look on it.
Commit removed.